### PR TITLE
Improves names of animation classes

### LINKED ISSUE #1170: rename animation_performance.dart to performance.dart
animation/animation_performance.dart is repetitive.

### PR DESCRIPTION
**Pull request by abarth**
Fixes #1170

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie @mpcomplete

File: sky/packages/sky/lib/src/animation/animated_value.dart [hunk 1]
```diff
@@ -19,9 +19,9 @@ enum Direction {
 ///
 /// AnimatedVariables, by convention, must be cheap to create. This allows them to be used in
```
Comment (mpcomplete) [inline sky/packages/sky/lib/src/animation/animated_value.dart]: fix comment

Comment (abarth) [inline sky/packages/sky/lib/src/animation/animated_value.dart]: Done.

**Comment by Hixie** [comment]
LGTM

File: sky/packages/sky/lib/src/animation/performance.dart [hunk 2]
```diff
@@ -51,30 +51,30 @@ abstract class WatchableAnimationPerformance {
 /// may also take direct control of the timeline by manipulating [progress], or
 /// [fling] the timeline causing a physics-based simulation to take over the
 /// progression.
-class AnimationPerformance implements WatchableAnimationPerformance {
-  AnimationPerformance({ this.duration, double progress }) {
+class Performance implements PerformanceView {
```
Comment (mpcomplete) [inline sky/packages/sky/lib/src/animation/performance.dart]: I have a small worry that it'll be easy to confuse this with e.g. CPU performance. Would it be better to shorten it as "Animation" instead? I don't feel that strongly, though.

**Comment by mpcomplete** [comment]
LGTM

File: sky/packages/sky/lib/src/animation/performance.dart (hunk 2 above)
Comment (Hixie) [inline sky/packages/sky/lib/src/animation/performance.dart]: Animation is way too vague, IMHO. I'm not a huge fan of Performance either, but I don't think there'd be that much confusion with performance. Maybe I'm not imaginative enough though. :-)

Comment (Hixie) [inline sky/packages/sky/lib/src/animation/performance.dart]: Specifically, the problem with Animation is that it is very easy to confuse with all the other terms in this space: Animatable, AnimatedValue, Animation... and what would you call ValuePerformance?

**Pull request merged**